src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,45 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    public static final int EMPTY_INDICATOR = -1;
+    private static final int MAX_SIZE = 12;
+    private int[] NUMBERS = new int[MAX_SIZE];
+    private static final int TOTAL = 11;
```

Comment:
This constant is related to MAX_SIZE. Changing only one leads to an error (inconsistency).